website/src/pages/download.mdx

```diff
@@ -0,0 +1,40 @@
+import { customFields } from '../../docusaurus.config.json'
+
+export const Link = () => {
+  const curr = `https://github.com/GMOD/jbrowse-components/releases/tag/${customFields.currentVersion}`
+  return <a href={curr}>{curr}</a>
+}
+
+# JBrowse 2 download
+
+To get started with JBrowse 2, we recommend using the "jbrowse CLI"
+
+Run
+
+```
+npm install -g @jbrowse/cli
+jbrowse create jbrowse_folder
+```
+
+This will download the latest jbrowse to a folder called jbrowse_folder
+
+If you have a server with apache or nginx, you can then
+
+```
+mv jbrowse_folder /var/www/html/jbrowse2
+```
+
+Then your instance will be available from http://localhost/jbrowse2
+
+If you use some other custom system, such as django, follow steps to host your
+content in your static folder
+
+Note: this folder is a "static website compatible" content, containing just JS,
+CSS, and no server side scripts, so Amazon s3 any static site hosting can be
+used as an alternative
+
+See [the quickstart guide](/docs/quickstart_web) for more info on loading tracks and getting started
+
+You can also manually download this release as a zip file from GitHub here
```

Comment:
A lot of this prose needs punctuation at the ends of statements, see https://reactjs.org/docs/create-a-new-react-app.html for example